spec/features/homescreen/robots_spec.rb

```diff
@@ -36,8 +36,9 @@
   end
 
   it 'disallows global paths and paths from public project' do
-    expect(page).to have_content('Disallow: /calendar')
     expect(page).to have_content('Disallow: /activity')
+    expect(page).to have_content('Disallow: /activities')
+    expect(page).to have_content('Disallow: /search')
 
     expect(page).to have_content("Disallow: /projects/#{project.identifier}/repository")
```

Comment:
Shouldn't `Disallow: /projects/#{project.identifier}/repository"` be there too?

Reply:
Can work by adding `Disallow: <%= search_path(project_id: p.identifier) %>` to `robots.text`
And also, shouldn't both the path with identifier and the path with id be disallowed?
```
Disallow: /projects/project-1/repository
Disallow: /projects/1/repository
Disallow: /projects/project-1/search
Disallow: /projects/1/search
Disallow: /projects/project-1/work_packages
Disallow: /projects/1/work_packages
Disallow: /projects/project-1/activity
Disallow: /projects/1/activity
```

Reply:
Good points. Changed as requested.